examples/wiseplat_live_strategy_print_OHLCV.py

```diff
@@ -0,0 +1,144 @@
+            self.days_back,
+            self.figi,
+        )
+        async for candle in self.client.get_all_candles(
+            figi=self.figi,
+            from_=now() - timedelta(days=self.days_back),
+            to=now(),
+            interval=self.timeframe,
+        ):
+            if candle not in self.candles:
+                if candle.is_complete:  # if candle is complete
+                    self.candles.append(candle)
+                    logger.debug("Found %s - figi=%s", candle, self.figi)
+
+    async def ensure_market_open(self):
+        """
+        Ensure that the market is open. Loop until the instrument is available.
+        :return: when instrument is available for trading
+        """
+        trading_status = await self.client.market_data.get_trading_status(
+            figi=self.figi
+        )
+        while not (
+            trading_status.market_order_available_flag
+            and trading_status.api_trade_available_flag
+        ):
+            logger.debug("Waiting for the market to open. figi=%s", self.figi)
+            await asyncio.sleep(60)
+            trading_status = await self.client.market_data.get_trading_status(
+                figi=self.figi
+            )
+
+    async def main_cycle(self):
+        """Main cycle for live strategy."""
+        while True:
+            try:
+                await self.ensure_market_open()
+                await self.get_historical_data()
+
+            except AioRequestError as are:
+                logger.error("Client error %s", are)
+
+            await asyncio.sleep(self.check_interval)
+
+    async def start(self):
+        """Strategy starts from this function."""
+        if self.account_id is None:
+            try:
+                self.account_id = (
+                    (await self.client.users.get_accounts()).accounts.pop().id
+                )
+            except AioRequestError as are:
+                logger.error("Error taking account id. Stopping strategy. %s", are)
+                return
+        await self.main_cycle()
+
+
+async def run_strategy(portfolio, timeframe, days_back, check_interval):
+    """From this function we are starting
+    strategy for every ticker from portfolio.
+    """
+    # pylint: disable=unnecessary-dunder-call
+    client = await AsyncClient(token=TOKEN, app_name="TinkoffApp").__aenter__()
+    for instrument in portfolio:
+        strategy = PrintOnlyCandlesStrategy(
+            figi=instrument,
+            timeframe=timeframe,
+            days_back=days_back,
+            check_interval=check_interval,
+            client=client,
+        )
+        asyncio.create_task(strategy.start())
+
+
+if __name__ == "__main__":
+
+    portfolio = {"BBG004730N88", "BBG004730ZJ9"}  # SBER, VTBR
+    timeframe = (
+        CandleInterval.CANDLE_INTERVAL_1_MIN
+    )  # ..._1_MIN,..._5_MIN, ..._15_MIN, ..._HOUR, ..._DAY
+    days_back = 1
+    check_interval = 10  # seconds to check interval for new completed candle
```

Comment:
```suggestion
    check_interval_period = timedelta(seconds=10)  # period between checking interval for new completed candle
```

а далее `check_interval_period.total_seconds()`

Reply:
ok

Reply:
зачем усложнять?

Reply:
Усложнять не надо, как по мне, оно читабельнее будет, и комменты можно будет убрать)

Reply:
Ну либо _seconds добавить в название переменной check_interval, неочевидно